GameServerLib/GameObjects/AttackableUnits/AI/Minion.cs

```diff
@@ -78,7 +47,10 @@ public void PauseAi(bool b)
         public override void OnAdded()
         {
             base.OnAdded();
-            _game.PacketNotifier.NotifyMinionSpawned(this, Team);
+            if (!(Owner == null))
```

Comment:
Why no owner is a problem? if it is because of how Notify implemented then put the null check there.

Reply:
Packet change if no owner so we need to check, resolved.